plenum/server/node.py

```diff
@@ -1243,7 +1243,7 @@ def select_primaries_if_needed(self, old_required_number_of_instances, txn_data=
             # We need to do a view change only during usual ordering. Because
             # if this is usual catchup, then, we will apply primaries from audit,
             # after catchup finish.
-            self.view_changer.on_node_count_changed()
+            self.master_replica.internal_bus.send(VoteForViewChange(Suspicions.NODE_COUNT_CHANGED))
```

Comment:
I think it may be convenient to have a helper method to trigger a view change.
Also I suspect that eventually these trigerring calls will be moved to services, won't they?

Reply:
I was thinking about creating a helper method, but decided not to do so because:
- this is just sending internal message, and main verbosity comes here from fact that node doesn't have internal bus - so it resorts to accessing internal bus of master replica
- eventually these triggering calls will be moved to some services, which will have internal buses, so creating helper method in node would be futile